lib/documents/schemas/esi_funds.json

```diff
@@ -180,7 +180,7 @@
     }
   ],
   "filter": {
-    "format": "european_structural_investment_fund"
+    "format": "esi_fund"
```

Comment:
We can't (easily) change this as this value is used in Search API too and is associated with all the ESI specialist documents.

Reply:
Yeah was worried that might be the case, format was a convenient value to use from the schema to set the document type in the test. I'm trying to add a conditional, in the test, to do this for but having some issues accessing DOM elements atm 😢

Reply:
I'm conceding defeat on esi_funds, at least as far as this PR goes. Wrangling this is holding up the rest of the PR, so i'll make a note of this in the PR description and add esi_funds to the exception list.